README.md

```diff
@@ -18,15 +17,25 @@ Step by step instructions on deploying this charm:
     juju add-relation nginx-proxy:template consul:consul-api
 
 
-# Configuration
+# Charm Configuration
 
 **service-name** is the name that consul-template builds the reverse proxy list
 from.
 
-# Contact Information
+### Reconfigure w/ auto forwarding from Consul
 
-Though this will be listed in the charm store itself don't assume a user will know that, so include that information here:
+The charm has a `public-website` relationship, that implements a prototype
+interface to setup automatic website proxy'ing based on data from Consul.
 
+    juju deploy cs:~zoology/trusty/consul
+    juju deploy cs:~zoology/trusty/dns
+    juju set dns provider=rt53 provider_keys="AWS_ACESS_KEY_ID|XXX AWS_PRIVATE ACCESS_KEY|XXX"
```

Comment:
AWS_ACCESS_KEY is misspelled here.  Also this is kind of a complex data structure to set using Juju.  You will need more explanation when to use spaces or pipes.